packages/@aws-cdk/aws-redshift/README.md

```diff
@@ -166,7 +166,7 @@ with the supplied schema.
 
 ```ts fixture=cluster
 new Table(this, 'Table', {
-  tableColumns: [{ name: 'col1', dataType: 'varchar(4)' }, { name: 'col2', dataType: 'float' }],
+  tableColumns: [{ id: 'col1', name: 'col1', dataType: 'varchar(4)' }, { id: 'col2', name: 'col2', dataType: 'float' }],
```

Comment:
why not make `id` optional and have it default to `name`?

Reply:
Potentially, that's what I was considering.
However, if we default it to the `name` attribute, wouldn't this just be issue that this PR is trying to tackle? (They change the name attribute, and accidently change the id)
I was thinking of making `name` optional and have it default to `id`?

Reply:
Actually, my mistake. Given that the `id` is optional, the above error would not happen. However, if the `name` attribute is not the original version, the column will be dropped. Could this be a desired implementation, from a developers perspective?

Reply:
This has been done, however I should note that the developer will now have to deploy twice if they decide to "re-use" the id system. We would have to let them know in the feature flag, if you would still like this to included. The below is the check behind this

```typescript
oldColumn.id ? oldColumn.id === column.id ?? column.name : oldColumn.name === column.name
```

Reply:
This has been implemented